src/main/java/com/cascadebot/cascadebot/events/CommandListener.java

```diff
@@ -60,6 +64,23 @@ public void onGuildMessageReceived(GuildMessageReceivedEvent event) {
     private void processCommands(GuildMessageReceivedEvent event, GuildData guildData, String trigger, String[] args, boolean isMention) {
         ICommandMain cmd = CascadeBot.INS.getCommandManager().getCommand(trigger, event.getAuthor(), guildData);
         if (cmd != null) {
+            if (cmd.getModule().isPublicModule() &&
+                    !guildData.isModuleEnabled(cmd.getModule())) {
+                if (guildData.willDisplayModuleErrors()) {
+                    EmbedBuilder builder = MessagingObjects.getClearThreadLocalEmbedBuilder();
+                    builder.setDescription(String.format("Module `%s` disabled!", cmd.getModule().toString()));
+                    builder.setTimestamp(Instant.now());
+                    builder.setFooter("Requested by " + event.getAuthor().getAsTag(), event.getAuthor().getEffectiveAvatarUrl());
+                    Messaging.sendDangerMessage(event.getChannel(), builder, guildData.getUseEmbedForMessages());
+                }
+                CascadeBot.logger.info(
```

Comment:
We should maybe add this to the modlog when we work on that but considering modules are a per guild thing I don't think logging to the console is the best

Reply:
Hmm, my thoughts were that it's bound to be less spammy than command logging that we already do. I'll remove it

Reply:
I mean it's spam for a thing that does go through. commands actually do something.